src/main.rs

```diff
@@ -101,6 +103,12 @@ enum Matchers {
     Myers,
 }
 
+#[derive(Debug, Eq, PartialEq, RustcDecodable)]
+enum Output {
+    NoOutput, // Default.
+    Json,
```

Comment:
I don't think the others should be: it's just that JSON really is an acronym in normal day-to-day life.